### PR TITLE
Grid layout

### PR DESCRIPTION
**Pull request by josepplloo**
## Summary

- grid redesigne added
## Screenshots

image

## Trello Card

https://trello.com/c/U1RCbBaM/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

File: maquetado/scss/layout.scss [hunk 1]
```diff
@@ -1,7 +1,15 @@
 .techs {
-  display: flex;
-  flex-wrap: wrap;
-  max-width: 100%;
-  margin: 70px auto 0;
-  width: 1024px;
+  align-content: space-around;
```
Comment (nmartinelli23) [inline maquetado/scss/layout.scss]: No need of this instruction.

File: maquetado/index.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
   </nav>
   <h1 class="title">Front-End Tools</h1>
   <div class="techs">
-    <div class="tech">
+    <div class="tech first">
```
Comment (henryzarza) [inline maquetado/index.html]: Change this
```suggestion
    <div class="tech">
```

File: maquetado/index.html [hunk 3]
```diff
@@ -73,7 +73,7 @@ <h6 class="tech-release">
   </div>
   <h1 class="title">Back-End Tools</h1>
   <div class="techs">
-    <div class="tech">
+    <div class="tech first">
```
Comment (henryzarza) [inline maquetado/index.html]: The same here

File: maquetado/scss/layout.scss [hunk 4]
```diff
@@ -1,7 +1,14 @@
...
-  margin: 70px auto 0;
-  width: 1024px;
+  display: grid;
+  grid-gap: 30px;
+  grid-template-columns: 190px 190px 190px;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.first {
```
Comment (henryzarza) [inline maquetado/scss/layout.scss]: What do you think about use `first-child`?

File: maquetado/scss/layout.scss [hunk 5]
```diff
@@ -1,7 +1,14 @@
...
+  border-right: 2px solid $green;
+  grid-row: 1 / span 3;
+  justify-items: center;
+  padding: 230px 20px;
```
Comment (henryzarza) [inline maquetado/scss/layout.scss]: Why this padding?

File: maquetado/scss/tech.scss [hunk 6]
```diff
@@ -1,31 +1,8 @@
...
-    width: 100%;
-    z-index: -1;
-  }
-
-  &:hover::after {
-    transform: translate(8px,-8px);
-  }
+  display: grid;
+  grid-template-columns: 1fr;
+  place-items: center;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Delete this

File: maquetado/scss/tech.scss [hunk 7]
```diff
@@ -1,31 +1,8 @@
...
-    transition: 0.5s;
-    width: 100%;
-    z-index: -1;
-  }
-
-  &:hover::after {
-    transform: translate(8px,-8px);
-  }
+  display: grid;
+  grid-template-columns: 1fr;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Why this? I think that is better if you define in the container
```suggestion
  grid-template-columns: repeat(3, 1fr);
```

File: maquetado/scss/layout.scss (hunk 4 above)
Comment (josepplloo) [reply maquetado/scss/layout.scss]: excelent idea!

**Pull request merged**